json-rpc/types/src/views.rs

```diff
@@ -621,9 +621,12 @@ pub struct ScriptView {
     // script code bytes
     #[serde(skip_serializing_if = "Option::is_none")]
     pub code: Option<BytesView>,
-    // script arguments, converted into string with type information.
+    // script arguments, converted into string with type information
     #[serde(skip_serializing_if = "Option::is_none")]
     pub arguments: Option<Vec<String>>,
+    // script function arguments, converted into hex encoded BCS bytes
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub arguments_bcs: Option<Vec<BytesView>>,
```

Comment:
please update the related document:
1. add new field to https://github.com/diem/diem/blob/main/json-rpc/docs/type_transaction.md#type-script, and update field `arguments` doc too.
2. add a change to https://github.com/diem/diem/blob/main/json-rpc/API-CHANGELOG.md
3. add related field to https://github.com/diem/diem/blob/main/json-rpc/types/src/proto/jsonrpc.proto, so that client uses it to generate code can have the new field.

Reply:
@xli  Changes added in 9639dcb. Feel free to let me know if the changes are OK :)